server/src/main/java/org/apache/iotdb/db/metadata/logfile/MLogWriter.java

```diff
@@ -366,6 +371,7 @@ public void operation(String cmd) throws IOException, MetadataException {
         createTimeseries(plan);
         break;
       case MetadataOperationType.CREATE_ALIGNED_TIMESERIES:
+      case MetadataOperationType.AUTO_CREATE_DEVICE_MNODE:
```

Comment:
Maybe add a "TODO" here for later polishing

Reply:
Since `AUTO_CREATE_DEVICE_MNODE` could never appear in the old mlog, so we do not need to support this case forever : )